py3status/modules/file_status.py

```diff
@@ -68,17 +111,40 @@ def file_status(self):
                 'cached_until': self.py3.CACHE_FOREVER,
             }
 
-        if exists(self.path):
-            icon = self.icon_available
-            color = self.py3.COLOR_GOOD
-        else:
-            icon = self.icon_unavailable
-            color = self.py3.COLOR_BAD
+        # init datas
+        paths = sorted([files for path in self.path for files in glob(path)])
+        paths_number = len(paths)
+
+        # get thresholds
+        if self.thresholds:
+            self.py3.threshold_get_color(paths_number, 'paths')
```

Comment:
(Request) Move this down right before the return (traditionally left at bottom for looping or such).